elements/pfe-navigation/src/pfe-navigation.html

```diff
@@ -55,5 +55,4 @@
 
 </nav>
 
-<!-- @todo: figure out to do the overlay in shadow DOM -->
-<!-- <div class="pfe-navigation__overlay"></div> -->
+<div class="pfe-navigation__overlay" hidden></div>
```

Comment:
TIL there's an attribute called hidden!

Reply:
@wesruv yeah, I learned about it fairly recently as well! It is a cool attribute.